testgrid/config/config.yaml

```diff
@@ -1995,6 +1995,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-gce-device-plugin-gpu
   - name: gce-gpu
     test_group_name: ci-kubernetes-e2e-gce-gpu
+  - name: gce-min-node-permissions
+    test_group_name: ci-kubernetes-e2e-gce-min-node-permissions
```

Comment:
you'll need to add something similar to https://github.com/ihmccreery/test-infra/blob/21a4014b4bd783ce58a7c93dee03a42624692bf6/testgrid/config/config.yaml#L1735-L1736 for the new job, and you should be good to go.

Reply:
Done.